### ISSUE TITLE
Reset STEPWAT2 branch to master

### ISSUE THREAD

**Issue by chaukap** [DrylandEcology/rSFSTEP2#154]
The branch has been changed to bugfix_337_reorder_species_input to test. Once bugfix_337_reorder_species_input is merged into master of STEPWAT2 we need to change the name on this branch.

**Comment by kpalmqui** [comment]
resolved by https://github.com/DrylandEcology/rSFSTEP2/commit/be94eec7acd44a6b6d2e5aa530b6386b4095128d
